src/main/resources/page/js/footer.js

```diff
@@ -0,0 +1,25 @@
+class FooterLinks {
+    constructor(links) {
+        this.links = links;
+        this.addFooterLinks(links);
+    }
+
+    addFooterLinks(links) {
+        for (let [linkName, href] of Object.entries(links)) {
+            this.createFooterLinks(linkName, href);
+        }
+    }
+
+    createFooterLinks(linkName, href) {
```

Comment:
What do you think, if we just return the FooterLink in this class...
And in the addFooterLinks method, we could append to this link to the footer (parent) element.

In this case, the consistency (in the header.js file when you wrote methods with "create" prefix, you always returned value) would be better.

Reply:
It's a good idea, thanks!